.travis.yml

```diff
@@ -67,38 +43,36 @@ jobs:
     - CC_VER="5"
     - PLATFORM='xenial'
     script: "./travis/build.sh"
-
-  ######################## XENIAL-EMSCRIPTEN
   - stage: build+test
     os: linux
     env:
     - PLATFORM='xenial-emscripten'
     script: "./travis/build.sh"
-
-#  ######################## OSX + TESTS
```

Comment:
Let's keep OSX there.. It has been disabled until Travis improves their osx build times